flink-scala/src/main/scala/org/apache/flink/api/scala/DataSet.scala

```diff
@@ -89,7 +89,7 @@ class DataSet[T: ClassTag](set: JavaDataSet[T]) {
   /**
    * Returns the TypeInformation for the elements of this DataSet.
    */
-  def getType(): TypeInformation[T] = set.getType()
```

Comment:
Now that I am looking at the code the majority of examples of exposing Java getters in the Scala api for both `DataStream` and `DataSet` are declared without paranthesis currently.